powerline/renderers/bar.py

```diff
@@ -1,45 +0,0 @@
-# vim:fileencoding=utf-8:noet
-from __future__ import (unicode_literals, division, absolute_import, print_function)
-
-from powerline.renderer import Renderer
-from powerline.colorscheme import ATTR_UNDERLINE
-
-
-class BarRenderer(Renderer):
-	'''bar (bar ain't recursive) renderer
-
-
-	See documentation of `bar <https://github.com/LemonBoy/bar>`_ and :ref:`the usage instructions <bar-usage>`
-	'''
-
-	character_translations = Renderer.character_translations.copy()
-	character_translations[ord('%')] = '%%'
```

Comment:
And in the message there are unicode characters: all spaces are being replaced with non-breaking spaces. This is not in the test code or in assertion error, but this is in stack trace which shows you the source code: https://github.com/ZyX-I/powerline/blob/59e66dca32f2bf17cb7cabec80b3c8d729795266/tests/test_configuration.py#L191.